spec/fixtures/CLOWNS_OF_AMERICA-iso19139.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <MD_Metadata xmlns="http://www.isotc211.org/2005/gmd" xmlns:gco="http://www.isotc211.org/2005/gco" xmlns:gts="http://www.isotc211.org/2005/gts" xmlns:srv="http://www.isotc211.org/2005/srv" xmlns:gml="http://www.opengis.net/gml" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <fileIdentifier>
-    <gco:CharacterString>edu.stanford.purl:vx812cc5548</gco:CharacterString>
+    <gco:CharacterString>edu.stanford.purl:vx813cc5549</gco:CharacterString>
```

Comment:
was there a reason for this change in druid?

Reply:
Yes, the same fixture was being used for 2 different and unrelated tests. This splits them into separate fixtures with separate druids.